Classes/Extensions/DDDispatchQueueLogFormatter.h

```diff
@@ -24,6 +24,11 @@
 #import "DDLog.h"
 
 
+typedef NS_OPTIONS(NSUInteger, DDDispatchQueueLogFormatterOptions) {
+    DDDispatchQueueLogFormatterShareable = 1 << 0,
+};
```

Comment:
I think we should have named options for both on and off. Usually the one that has a raw value of 0 will be default.